client/stylesheets/_toCat.scss

```diff
@@ -5792,6 +5792,10 @@ textarea {
     margin-left: 30px;
 }
 
+.helpHide {
+    display: none !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces